requirements.txt

```diff
@@ -4,4 +4,6 @@ flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 django-debug-toolbar==3.2.4
-django-crispy-forms==1.14.0
\ No newline at end of file
+django-crispy-forms==1.14.0
+django-bootstrap-icons==0.8.7
+crispy-bootstrap4
```

Comment:
better to specify version in your requirements. You can use pip freeze to generate requirements.txt file